actions/databases.go

```diff
@@ -294,6 +300,33 @@ func DatabasesPut(c buffalo.Context) error {
 		log.Println("Modified RDS instance", instanceOutput)
 	}
 
+	if input.Tags != nil {
+		log.Println("Updating tags for "+c.Param("db")+"\n", input.Tags)
```

Comment:
dont think you need the `+"\n"` for `log.Println`

Reply:
I added it because I wanted the tags printed on the next line

Reply:
ah, I missed that.  even still, its a log message, i dont really think you want that.